migrations/tenant/0010-note-relations@add-note-relations-table.sql

```diff
@@ -11,9 +11,9 @@ CREATE TABLE IF NOT EXISTS public.note_relations (
 --
 -- Name: note_relations relation_id; Type: PK CONSTRAINT; Schema: public; Owner: codex
 --
-ALTER TABLE public.note_relations DROP CONSTRAINT IF EXISTS relation_id_pkey;
+ALTER TABLE public.note_relations DROP CONSTRAINT IF EXISTS note_relations_pkey;
 ALTER TABLE public.note_relations 
-    ADD CONSTRAINT relation_id PRIMARY KEY (id);
+    ADD CONSTRAINT note_relations_pkey PRIMARY KEY (id);
```

Comment:
you should not modify existing migration script, it will break `yarn migrate` at others machines

Reply:
I've just picked all the changes from recent migration update. This file is the same we have in the main branch